### PR TITLE
elenamelnik homework 3 

### PR DESCRIPTION
**Pull request by Elena983**

### CODE REVIEW

File: elenamelnik/hw3/task1.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+x = 5
+y = 10
+x = x * y
+y = x / y
+x = x / y
```
Comment (katemalash) [inline elenamelnik/hw3/task1.py:5]: Good decision.

File: elenamelnik/hw3/task2.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+b = int(input('Please enter \n'))
+kb = b * 2**10
```
Comment (katemalash) [inline elenamelnik/hw3/task2.py:2]: Are you sure it's correct?

File: elenamelnik/hw3/task2.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+mb = b * 2**20
```
Comment (katemalash) [inline elenamelnik/hw3/task2.py:3]: Are you sure it's correct?

File: elenamelnik/hw3/task3.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+w=float(input("Width is \n"))
+h=float(input('Height is \n'))
+s=w*h
+P=2*(w + h)
+print("s = ", s)
+print("P = ", P)
```
Comment (katemalash) [inline elenamelnik/hw3/task3.py:6]: Good Decision

File: elenamelnik/hw3/task4.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+r=float(input("Radius is \n"))
+s=3.14*r**2
+P=2*3.14*r
+print("s = ", s)
+print("P = ", P)
```
Comment (katemalash) [inline elenamelnik/hw3/task4.py:5]: Good Decision.

File: elenamelnik/hw3/task5.py [hunk 6]
```diff
@@ -0,0 +1,6 @@
+import math
+print("Lengths of triangle sides \n")
+a = float(input("a = "))
+b = float(input("b = "))
+c = math.sqrt(a**2 + b**2)
+print("The length of the hypotenuse is", c )
```
Comment (katemalash) [inline elenamelnik/hw3/task5.py:6]: Good Decision.

File: elenamelnik/hw3/task6.py [hunk 7]
```diff
@@ -0,0 +1,2 @@
+cats = input('Collection cats')
+cats_names = print('Siberian_cat\n', 'russian_cat\n', 'grayish_cat\n')
```
Comment (katemalash) [inline elenamelnik/hw3/task6.py:2]: It would be better to use tuple, for example

**Review by katemalash — APPROVED** [review]
In general, home work is done good. You can merge it after fixing the minor issues.

File: elenamelnik/hw3/task2.py (hunk 2 above)
Comment (Elena983) [reply elenamelnik/hw3/task2.py:2]: And what's wrong?

File: elenamelnik/hw3/task6.py (hunk 7 above)
Comment (Elena983) [reply elenamelnik/hw3/task6.py:2]: can you give me an example with plural form?

**Pull request merged**